Unsupported YUM repo when using MysqL 8.0

This is a subtask of #128 which is also listing some extra repos.

Using a CentOS 7 server with MySQL 8.0 installed:
```
/usr/local/cpanel/bin/whmapi1 start_background_mysql_upgrade version=8.0
```

The elevate script detects some blockers, and complains about
`mysql80-community`, `mysql-connectors-community` and `mysql-tools-community` repos.

These three repos are defined by '/etc/yum.repos.d/mysql-community.repo'


```
[root@10-2-68-222 ~]# /scripts/elevate-cpanel --check
* 07-22:05:07 (315) [INFO] Successfully verified signature for cpanel (key types: release, development).
* 07-22:05:07 (2504) [WARN] *** Elevation Blocker detected: ***
This installation of cPanel (11.107.9999.94) does not appear to be up to date. Please upgrade cPanel to a most recent version.

* 07-22:05:16 (3018) [ERROR] 9 package(s) installed from unsupported YUM repo 'mysql80-community' from /etc/yum.repos.d/mysql-community.repo
* 07-22:05:16 (3029) [WARN] Unsupported YUM repo enabled 'mysql-connectors-community' without packages installed from /etc/yum.repos.d/mysql-community.repo
* 07-22:05:16 (3029) [WARN] Unsupported YUM repo enabled 'mysql-tools-community' without packages installed from /etc/yum.repos.d/mysql-community.repo
* 07-22:05:16 (2504) [WARN] *** Elevation Blocker detected: ***
One or more enabled YUM repo are currently unsupported.
You should disable these repositories and remove packages installed from them
before continuing the update.

Consider reporting this limitation to https://github.com/cpanel/elevate/issues


* 07-22:05:25 (2925) [INFO] Checking if your system is up to date:
* 07-22:05:25 (3361) [INFO] Running: /usr/bin/yum clean all
* 07-22:05:25 (3362) [INFO]
* 07-22:05:25 (3372) [INFO] Loaded plugins: fastestmirror, universal-hooks
* 07-22:05:25 (3372) [INFO] Cleaning repos: EA4 cpanel-addons-production-feed cpanel-plugins base epel
* 07-22:05:25 (3372) [INFO]               : extras imunify360 imunify360-rollout-1 imunify360-rollout-2
* 07-22:05:25 (3372) [INFO]               : imunify360-rollout-3 imunify360-rollout-4
* 07-22:05:25 (3372) [INFO]               : mysql-connectors-community mysql-tools-community
* 07-22:05:25 (3372) [INFO]               : mysql80-community updates wp-toolkit-cpanel
* 07-22:05:25 (3372) [INFO]               : wp-toolkit-thirdparties
* 07-22:05:26 (3372) [INFO] Cleaning up list of fastest mirrors
* 07-22:05:26 (3372) [INFO] Other repos take up 19 M of disk space (use --verbose for details)
* 07-22:05:26 (3382) [INFO]
* 07-22:05:26 (3361) [INFO] Running: /usr/bin/yum check-update
* 07-22:05:26 (3362) [INFO]
* 07-22:05:26 (3372) [INFO] Loaded plugins: fastestmirror, universal-hooks
* 07-22:05:26 (3372) [INFO] Determining fastest mirrors
* 07-22:05:29 (3372) [INFO]  * EA4: 184.94.196.92
* 07-22:05:29 (3372) [INFO]  * cpanel-addons-production-feed: 184.94.196.92
* 07-22:05:29 (3372) [INFO]  * cpanel-plugins: 184.94.196.92
* 07-22:05:29 (3372) [INFO]  * epel: repos.eggycrew.com
* 07-22:05:42 (3382) [INFO]
* 07-22:05:42 (2932) [INFO] Checking /scripts/sysup
* 07-22:05:42 (3361) [INFO] Running: /scripts/sysup
* 07-22:05:42 (3362) [INFO]
* 07-22:05:43 (3372) [INFO] checkyum version 22.3  (excludes: bind-chroot)
* 07-22:05:44 (3372) [INFO]   All Needed Packages are already installed.
* 07-22:05:44 (3382) [INFO]
* 07-22:05:44 (2804) [INFO] Checking EasyApache profile compatibility with Almalinux 8.
* 07-22:05:44 (1448) [INFO] Running: /usr/local/bin/ea_current_to_profile --target-os=AlmaLinux_8
* 07-22:05:45 (1469) [INFO] Backed up EA4 profile to /etc/cpanel/ea4/profiles/custom/current_state_at_2022-10-07_22:05:45_modified_for_AlmaLinux_8.json
* 07-22:05:47 (570) [WARN] Please fix the detected issues before performing the elevation process.

```

Suggested fix: add `mysql-community.repo` to the `DISABLE_MYSQL_YUM_REPOS` list

--- a/changelog
+++ b/changelog
@@ -1,9 +1,9 @@
-
-2020-10-10 - version 1
+{{ NEXT }} - version 1
 User Facing
  - Fix run_once entry names in stage file
  - Work around use of /boot/grub/ in some providers
  - Implement self-update mechanism for elevate-cpanel
+ - Allow upgrade when using MySQL from mysql-community.repo
 Internal:
  - read_stage_file() can read specific data
 2022-10-07 - version 0

--- a/elevate-cpanel
+++ b/elevate-cpanel
@@ -217,6 +217,8 @@ use constant DISABLE_MYSQL_YUM_REPOS => qw{
   MariaDB103.repo
   MariaDB105.repo
   MariaDB106.repo
+
+  mysql-community.repo
 };
 
 use constant VETTED_YUM_REPO => qw{
@@ -328,7 +330,7 @@ sub do_update ($self) {
         my $running_from = Cwd::abs_path($0) // '';
         if ( File::Copy::move( $temp_file, $running_from ) ) {
             chmod 0700, $running_from;
-            my $confirmed_version = Cpanel::SafeRun::Simple::saferunnoerror($running_from, '--version');
+            my $confirmed_version = Cpanel::SafeRun::Simple::saferunnoerror( $running_from, '--version' );
             chomp $confirmed_version;
             INFO("Script update to version $confirmed_version successful.");
         }

